docs/tools/function-tools.md

```diff
@@ -160,6 +160,16 @@ This tool is designed to help you start and manage tasks that are handled outsid
 
 When using a `LongRunningFunctionTool`, your function can initiate the long-running operation and optionally return an **initial result**, such as a long-running operation id. Once a long running function tool is invoked the agent runner pauses the agent run and lets the agent client to decide whether to continue or wait until the long-running operation finishes. The agent client can query the progress of the long-running operation and send back an intermediate or final response. The agent can then continue with other tasks. An example is the human-in-the-loop scenario where the agent needs human approval before proceeding with a task.
 
+### Pause and Resume Mechanism
```

Comment:
LongRunningFunctions are not the focus of the Resume feature, as such these doc change recommendations are off the mark and possibly misleading.